proto/commonfate/access/v1alpha1/access_request.proto

```diff
@@ -40,26 +45,39 @@ service AccessRequestService {
   rpc CloseAccessRequest(CloseAccessRequestRequest) returns (CloseAccessRequestResponse) {}
 }
 
+enum RequestStatus {
+  REQUEST_STATUS_UNSPECIFIED = 0;
+
+  REQUEST_STATUS_PENDING = 1;
+
+  REQUEST_STATUS_ACTIVE = 2;
+
+  REQUEST_STATUS_CLOSED = 3;
+}
+
 message QueryAccessRequestsRequest {
   // The token for the next page.
   string page_token = 1;
   bool archived = 2;
   optional commonfate.entity.v1alpha1.Order order = 3;
-  optional commonfate.entity.v1alpha1.EID requested_by = 4;
```

Comment:
Any reason to remove this? This will still be useful for building filters over all access requests